aspnetcore/fundamentals/middleware/index.md

```diff
@@ -252,7 +252,9 @@ Middleware is generally encapsulated in a class and exposed with an extension me
 
 [!code-csharp[](index/snapshot/Culture/StartupCulture.cs?name=snippet1)]
 
-The preceding sample code is used to demonstrate creating a middleware component. For ASP.NET Core's built-in localization support, see <xref:fundamentals/localization>.
+In the preceding code, the call `return next();` is equivalent to `return next(context);`.  The The `httpContext` is captured inside the next `Func`. See the [source code](https://github.com/aspnet/HttpAbstractions/blob/master/src/Microsoft.AspNetCore.Http.Abstractions/Extensions/UseExtensions.cs#L27) for more information.
```

Comment:
* "The The"
* `httpContext` should be `HttpContext`
* Since this isn't much more than an implementation nuance (and confusing without more explanation than this), I wouldn't resolve the issue. I'd have closed it as a wont-fix.
* WRT this approach, I'd recommend dropping the "See the code" sentence ...
  * Source code links are a pit of failure:
    * This one points to the archived repo.
    * Even pointing to the new repo, the link will probably eventually break, especially with a line number.
    * The link won't be current version-to-version without a manual update.
  * The source code doesn't explain out what's going on.